yml/microsoft/built-in/wbemcomn.yml

```diff
@@ -0,0 +1,18 @@
+---
+Name: wbemcomn.dll
+Author: v1stra
+Created: 2024-12-12
+Vendor: Microsoft
+ExpectedLocations: # No trailing slashes
+  - '%SYSTEM32%'
+  - '%SYSWOW64%'
+VulnerableExecutables:
+  - Path: 'C:\Windows\System32\Wbem\wbemcomn.dll'
```

Comment:
Hi @v1stra , thanks for opening this PR. Could you update this line to reference the executable it affects, rather than the DLL itself? thanks!